airbyte-db/db-lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -138,6 +138,9 @@ create table "public"."state"(
   "state" jsonb null,
   "created_at" timestamptz(35) not null default null,
   "updated_at" timestamptz(35) not null default null,
+  "stream_name" text null,
+  "namespace" text null,
+  "type" state_type null default null,
```

Comment:
should this be not null? might just be an idiosyncrasy of the dump and an actual problem though.